src/mlpack/methods/fastmks/fastmks_impl.hpp

```diff
@@ -301,6 +372,20 @@ void FastMKS<KernelType, MatType, TreeType>::Search(
     arma::Mat<size_t>& indices,
     arma::mat& kernels)
 {
+	Search(*queryTree, k, indices, kernels);
```

Comment:
The spacing is off here, the call to `Search()` should be indented only two spaces from the opening brackets.